services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+from django.db.models import QuerySet
```

Comment:
First should be build-in imports, then third-party and then custom. all groups separated by blank line